### PR TITLE
Set extra.executed_at field of transactions

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Set extra.executed_at field of transactions when executed
- Use extra.executed_at field in triggered functions 
- Save executed_at field in transaction pool
- Convert transactions to an executable form (i.e., instance of 'Transaction') before being executed
- Disable console logging by default to see clean messages of test cases
- Remove unused code for batch transaction
- Remove some redundant parameters

Testing:
- Ran tests:
```
yarn run test_unit
yarn run test_integration
```
- Reported broken test: https://github.com/ainblockchain/ain-blockchain/issues/291 

#287

### CODE REVIEW

File: db/functions.js [hunk 1]
```diff
@@ -426,7 +431,8 @@ class Functions {
     lastTxPath.push(lastLabel);
 
     return this.setValueOrLog(
-        ChainUtil.formatPath(lastTxPath), { tx_hash: transaction.hash }, auth, timestamp);
+        ChainUtil.formatPath(lastTxPath), { tx_hash: transaction.hash }, auth, timestamp,
+        transaction);
```
Comment (liayoo) [inline db/functions.js:435]: spaces?

File: db/index.js [hunk 2]
```diff
@@ -597,38 +597,6 @@ class DB {
     return ret;
   }
 
-  batch(txList) {
```
Comment (liayoo) [inline db/index.js:600]: Nice! 👍 Didn't even notice we weren't using this..!

File: node/index.js [hunk 3]
```diff
@@ -353,16 +353,18 @@ class BlockchainNode {
       return ChainUtil.logAndReturnError(
           logger, 1, `[${LOG_HEADER}] Blockchain node is NOT in SERVING mode: ${this.state}`, 0);
     }
-    const result = this.executeOrRollbackTransaction(tx);
+    const executableTx = Transaction.toExecutable(tx);
+    const result = this.executeOrRollbackTransaction(executableTx);
     if (ChainUtil.transactionFailed(result)) {
-      logger.info(`[${LOG_HEADER}] FAILED TRANSACTION: ${JSON.stringify(tx, null, 2)}\n ` +
+      logger.info(
```
Comment (liayoo) [inline node/index.js:359]: Could we actually change this to debug level?

**Review by liayoo — APPROVED** [review]
LGTM with some minor comments. Thanks!

File: db/functions.js (hunk 1 above)
Comment (platfowner) [reply db/functions.js:435]: could you be more specific?

File: node/index.js (hunk 3 above)
Comment (platfowner) [reply node/index.js]: Just tried to use enableRichTransactionLogging feature flag. PTAL

**Comment by platfowner** [comment]
Comments address, PTAL @liayoo

File: db/functions.js (hunk 1 above)
Comment (liayoo) [reply db/functions.js:435]: Ah, I meant the indentation seemed off for line 435.

File: node/index.js (hunk 3 above)
Comment (liayoo) [reply node/index.js]: Looks good! Thank you!

**Comment by liayoo** [comment]
Thanks for addressing the comments & sorry for my vague comment above 😅 LGTM!

File: db/functions.js (hunk 1 above)
Comment (platfowner) [reply db/functions.js:435]: Do you mean line wrapping like below?
style A:
```
  return someFunction(
      parameterA, parameterB, parameterC,
          parameterD);
````

I think some other styles like below are considered acceptable by many people:

style B:
```
  return someFunction(
      parameterA, parameterB, parameterC,
      parameterD);
```

style C:
```
  return someFunction(
      parameterA,
      parameterB,
      parameterC,
      parameterD);
```

Style B seems to be more common than style A in C++ or java world from my previous experience. 
Please let me know it was not the case (line wrapping with 100 char limit).

Comment (liayoo) [reply db/functions.js:435]: I see, thanks for the explanation. Style B actually seems reasonable to me as well (style A could waste a lot of space..). Let's just leave this as it is. I'll keep this style in mind!

**Comment by platfowner** [comment]
@liayoo Thanks for the fast review! Now merging..

**Pull request merged**